src/__tests__/custom-document/__fixtures__/special-extension/pages/_document.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Document, { Html, Head, Main, NextScript } from 'next/document';
+
+class CustomDocument extends Document {
+  render() {
+    return (
+      <>
```

Comment:
I'm not sure if omitting `<Html>` is valid. Have you tried using such document in Next?